resources/random.go

```diff
@@ -0,0 +1,13 @@
+package rps
+
+import (
+	"math/rand"
+)
+
+func Computer() string { //called by main, generates computer's input
+
+	rps := []string{"rock", "paper", "scissors"}
+
+	C := (rps[rand.Intn(len(rps))])
+	return C
```

Comment:
Not that this is wrong but you can reduce an allocation if you just return directly.


```suggestion
	return rps[rand.Intn(len(rps))]
```